index.html

```diff
@@ -0,0 +1,211 @@
+            <li><a href="#">Css</a></li>
+            <li><a href="#">Javascript</a></li>
+            <li><a href="#">Ruby</a></li>
+           
+           </ul>
+           <button class="work-see-button">See Project</button>
+        </article>
+      </div>
+    </section>
+
+    <!--work card section start-->
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
+        <div class="description">A daily selections of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+        <ul class="card-ul">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+        <button class="card-button">See Project</button>
+      </div>
+    </section>
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
+        <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+        <ul class="card-ul">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+        <button class="card-button">See Project</button>
+      </div>
+    </section>
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
+        <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+        <ul class="card-ul">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+        <button class="card-button">See Project</button>
+      </div>
+    </section>
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
+        <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+        <ul class="card-ul">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+        <button class="card-button">See Project</button>
+      </div>
+    </section>
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
+        <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+        <ul class="card-ul">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+        <button class="card-button">See Project</button>
+      </div>
+    </section>
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
+        <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
```

Comment:
I suggest you change this from a `<div>` to a `<p>`. Using correct and semantic html elements ensures that your app is friendly to screen readers.